react/week3/todo-list/src/App.js

```diff
@@ -1,17 +1,74 @@
 import "./App.css";
-import ToDo from "./Components/Todo";
-import ToDoList from "./Components/TodoList";
-import ToDoForm from "./Components/TodoForm";
-import { useState } from "react";
+import TodoList from "./Components/TodoList";
```

Comment:
Minor comment here but the components themselves, as you've done, should be CamelCase but folders are usually lower/kebab case _/components/_